integreat_cms/cms/templates/pois/poi_form_sidebar/barrier_free_box.html

```diff
@@ -0,0 +1,18 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    smile
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Barrier free" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div>
+        <label for="{{ poi_form.barrier_free.id_for_label }}" class="mt-0">{{ poi_form.barrier_free.label }}</label>
+        {% render_field poi_form.barrier_free|add_error_class:"border-red-500" %}
+        <div class="help-text">{{ poi_form.barrier_free.help_text }}</div>
```

Comment:
```suggestion
        <div class="help-text">{{ poi_form.barrier_free.help_text }}</div>
        {% render_field poi_form.barrier_free|add_error_class:"border-red-500" %}
```
Hmm, maybe we don't need to print "barrier free" twice? (Once as box header and once as label)